src/extension.ts

```diff
@@ -523,6 +523,37 @@ export function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(vscode.commands.registerCommand(commands.CommandsConsts.globalImportSnippets,
         async _ => handleCommand(() => commands.importSnippets(snippetsProvider))
     ));
+
+    context.subscriptions.push(vscode.languages.registerDocumentDropEditProvider({ language: '*', scheme: '*'}, new class DropEditorProvider implements vscode.DocumentDropEditProvider {
```

Comment:
instead of `{ language: '*', scheme: '*'}`, you can simply use `'*'`

Reply:
Done.